src/utils/destroy-backbone-view.js

```diff
@@ -11,6 +12,7 @@ export default function destroyBackboneView(view) {
     triggerMethodOn(view, 'before:detach', view);
   }
 
+  dom.removeEl(view.el);
```

Comment:
I actually couldn't figure out how to handle this.
I could leave this line out and let plain Backbone.View do it's own thing if the user decides to use it as is.

I couldn't decide to either override the plain view's `_removeElement` but that should be left up to the user.